.github/workflows/production-deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy To Production
+
+on:
+  workflow_dispatch:
```

Comment:
instead of 2 separate files we can use workflow inputs:
```
  workflow_dispatch:
    inputs:
      environment:
        type: environment
        description: Select the environment
      boolean:
        type: boolean
        description: True or False
      choice:
        type: choice
        description: Make a choice
        options:
        - foo
        - bar
        - baz
```